packages/layout/src/components/panel/es-sized-panel/es-sized-panel.tsx

```diff
@@ -48,7 +48,7 @@ export class SizedPanel {
 
     disconnectedCallback() {
         this.resizeObserver.disconnect();
-        sidebarWidth.reset();
+        this.cssVar.reset();
```

Comment:
Calling `reset()` doesn't seem to remove the handler from `window[LAYOUT_OBSERVERS]` is that ok?